File: basics/26-third-node/Solution.cpp
struct ListNode {
    int val;
    ListNode* next;
    ListNode(int x) : val(x), next(nullptr) {}
};

ListNode* getThird(ListNode* head) {
    if (!head || !head->next || !head->next->next) return nullptr;
    return head->next->next;
}

File: basics/26-third-node/Solution.py
class ListNode:
    def __init__(self, val=0, next=None):
        self.val = val
        self.next = next

def get_third(head: ListNode) -> ListNode:
    if not head or not head.next or not head.next.next:
        return None
    return head.next.next
